specification/iothub/resource-manager/Microsoft.Devices/stable/2021-07-01/iothub.json

```diff
@@ -0,0 +1,4401 @@
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "readOnly": true
+        },
+        "certificate": {
+          "description": "The certificate content",
+          "type": "string"
+        }
+      }
+    },
+    "CertificatePropertiesWithNonce": {
+      "description": "The description of an X509 CA Certificate including the challenge nonce issued for the Proof-Of-Possession flow.",
+      "type": "object",
+      "properties": {
+        "subject": {
+          "description": "The certificate's subject name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "expiry": {
+          "description": "The certificate's expiration date and time.",
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "readOnly": true
+        },
+        "thumbprint": {
+          "description": "The certificate's thumbprint.",
+          "type": "string",
+          "readOnly": true
+        },
+        "isVerified": {
+          "description": "Determines whether certificate has been verified.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "created": {
+          "description": "The certificate's create date and time.",
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "readOnly": true
+        },
+        "updated": {
+          "description": "The certificate's last update date and time.",
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "readOnly": true
+        },
+        "verificationCode": {
+          "description": "The certificate's verification code that will be used for proof of possession.",
+          "type": "string",
+          "readOnly": true
+        },
+        "certificate": {
+          "description": "The certificate content",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "IotHubProperties": {
+      "description": "The properties of an IoT hub.",
+      "type": "object",
+      "properties": {
+        "authorizationPolicies": {
+          "description": "The shared access policies you can use to secure a connection to the IoT hub.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SharedAccessSignatureAuthorizationRule"
+          }
+        },
+        "disableLocalAuth": {
+          "description": "If true, SAS tokens with Iot hub scoped SAS keys cannot be used for authentication.",
+          "type": "boolean"
+        },
+        "disableDeviceSAS": {
+          "description": "If true, all device(including Edge devices but excluding modules) scoped SAS keys cannot be used for authentication.",
+          "type": "boolean"
+        },
+        "disableModuleSAS": {
+          "description": "If true, all module scoped SAS keys cannot be used for authentication.",
+          "type": "boolean"
+        },
+        "restrictOutboundNetworkAccess": {
```

Comment:
publicNetworkAccess is a string enum below which is more in line with recommended patterns.  Can this not be "outboundNetworkAccess": "Restricted"/"Unrestricted"?

Reply:
@pilor this is as per the ARM guidelines.

Reply:
Do you have a link to the source for that?

Reply:
yes. I have sent you the link to your chat in teams

Reply:
Approvins as-is since this is a EP driven change.  Will ensure API review is involved before future changes are pushed to RPs